cmd/skopeo/sync.go

```diff
@@ -723,6 +726,15 @@ func (opts *syncOptions) run(args []string, stdout io.Writer) (retErr error) {
 		logrus.Warn("Running in dry-run mode")
 	}
 
+	var digestFile *os.File
+	if opts.digestFile != "" && !opts.dryRun {
+		digestFile, err = os.OpenFile(opts.digestFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+		if err != nil {
+			return fmt.Errorf("Error creating digest file: %w", err)
+		}
+		defer digestFile.Close()
```

Comment:
For writes, I think we want to also check for errors here (so that e.g. failures updating the directory or the inode with the final data are reported).

Reply:
Still outstanding. Why was this marked as resolved?

Reply:
not sure if understood the ask here @mtrmac

Reply:
If `digestFile.Close` fails, that should be detected and treated as a failure.

Reply:
Done, now handled correctly.